### DISCUSSION TITLE
Global Agent without custom server.

### DISCUSSION THREAD

**Discussion by amesas** [vercel/next.js#37391] [Help]
Right now we are using a custom server just to load https://github.com/gajus/global-agent and proxy some requests.

Like this: https://github.com/vercel/next.js/issues/20837

Is there a way to move this to NextJS and totally skip a custom server? With middlewares we have been able to remove all our custom server logic, except this one.

Thnx in advance for any help / ideas.

**Comment by aksjer** [comment]
> Right now we are using a custom server just to load https://github.com/gajus/global-agent and proxy some requests.
> 
> Like this: #20837
> 
> Is there a way to move this to NextJS and totally skip a custom server? With middlewares we have been able to remove all our custom server logic, except this one.
> 
> Thnx in advance for any help / ideas.

node -r global-agent/bootstrap node_modules/next/dist/bin/next dev
